src/Http/Http.Extensions/gen/StaticRouteHandlerModel/StaticRouteHandlerModel.Emitter.cs

```diff
@@ -68,7 +65,7 @@ public static string EmitRequestHandler(this Endpoint endpoint)
                     {{handlerSignature}}
                     {
                         {{setContentType}}
-                        {{resultAssignment}}{{awaitHandler}}handler();
+                        {{resultAssignment}}{{awaitHandler}}handler({{endpoint.EmitArgumentList()}});
```

Comment:
Probably something that will need to change when I start working on the parsing, because this list of arguments will need to be mapped to a set of local variables which are injected beforehand to store the results of the appropriate ```TryParse(...)``` operation.